Jenkinsfile

```diff
@@ -1,15 +1,11 @@
-pipeline {
-    agent { label 'docker' }
-    options {
-        disableConcurrentBuilds()
-    }
-    stages {
-        stage('Build') {
-            steps {
-                script {
-                  echo "This is an application build"
-                }
-            }
-        }
-    }
+library(
+        identifier: 'jenkins-shared@LEV-78-extend-jenkinsfile',
```

Comment:
Should this be pointing to `main`?
If there's a PR to get that jenkins-shared repo into `main`, then apologies in advance!